src/CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ set(LIB_SRCS
     "type.cpp"
     "type.h"
     "utils.cpp"
-    "utils.h")
```

Comment:
New files names need to be in quotes and in alphabetical order.

Reply:
Fixed